openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -164,6 +167,12 @@ func (e *extension) Reconcile(ctx context.Context, comp base.KComponent) error {
 		common.ConfigureIfUnset(&ks.Spec.CommonSpec, monitoring.ObservabilityCMName, monitoring.ObservabilityBackendKey, "none")
 	}
 
+	cleanDomainMappingOnce.Do(func() {
+		if err := e.cleanupDomainMapping(ctx, ks); err != nil {
+			return
```

Comment:
With that we loose the error, right? Is there a logger on the context?

Reply:
Update with at least logging the error.